private/rules/v2_lock_file.bzl

```diff
@@ -43,11 +43,20 @@ def _compute_lock_file_hash(lock_file_contents):
     return hash(repr(to_hash))
 
 def _to_m2_path(unpacked):
-    path = "{group}/{artifact}/{version}/{artifact}-{version}".format(
-        artifact = unpacked["artifactId"],
-        group = unpacked["groupId"].replace(".", "/"),
-        version = unpacked["version"],
-    )
+    version = unpacked["version"]
```

Comment:
This doesn't appear to be used.

Reply:
Heh, leftover from previous code that did exactly what you are asking for below about unifying the branches. Fixing both...